web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v447/migrate-default.sql

```diff
@@ -6,3 +6,5 @@ INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system
 
 ALTER TABLE Sources ADD COLUMN datahubEnabled BOOLEAN DEFAULT FALSE;
 ALTER TABLE Sources ADD COLUMN datahubConfiguration TEXT DEFAULT '';
+
+INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('metadata/delete/enablebackup', 'true', 2, 12012, 'n');
```

Comment:
Don't you also need an entry in /web/src/main/webapp/WEB-INF/classes/setup/sql/data/data-db-default.sql ?

Reply:
I have added such entry